examples/install/cluster-operator/07-crd-kafka.yaml

```diff
@@ -733,6 +733,9 @@ spec:
                   type: object
                   description: The Prometheus JMX Exporter configuration. See https://github.com/prometheus/jmx_exporter
                     for details of the structure of this configuration.
+                stunnelImage:
```

Comment:
Both here and in description ... wouldn't `tlsSidecarImage` or something like that be better? Does everyone know what is `stunnel`?